src/reqstool/resources/schemas/v1/requirements.schema.json

```diff
@@ -87,11 +79,7 @@
                     "description": "Needed in case the document has extenal references like NMB2B for example, this url might point to a source of information about the EuroControl requirements"
```

Comment:
extenal -> external

Do not use terms mainly internal to LFV, i.e. remove NMB2B and EuroControl. This is a generic tool although it is developed and used by LFV.